IntegratingSDK/IntegratingSDK/Restaurant.m

```diff
@@ -0,0 +1,34 @@
+//
+//  Restaurant.m
+//  IntegratingSDK
+//
+//  Created by Joe Susnick on 4/3/19.
+//  Copyright © 2019 facebook. All rights reserved.
+//
+
+#import "Restaurant.h"
+@import FacebookCore;
+
+@implementation Restaurant
+
+-(FBRestaurant *)openRestaurant {
+    FBEmployeeClassification *payrollEntry = [[FBEmployeeClassification alloc] initWithCaseValue:[FBEmployeeClassificationManager new]];
+
+    FBRestaurant *restaurant = [[FBRestaurant alloc] initWithName: @"Mama Cass's Sandwich Shack"
+                                                         specials: @[@"Ham sammiches"]
+                                                      regularMenu: @[@"PB&Js"]
+                                                         employee: [[FBEmployee alloc] initWithClassification: payrollEntry]
+                                                          payroll: payrollEntry
+                                                     registerCode: @"123"
+                                                     bathroomCode: @"345"];
+
+    [restaurant ]
```

Comment:
What is this?

Reply:
Poor naming on my part. Fixing. Good catch!